app/views/assessments/_gradesheet.html.erb

```diff
@@ -25,8 +25,7 @@
 <thead>
     <tr class="sortable">
         <th class="enumerator">#</th>
-        <th class="id">Email</th>
-        <th>Name</th>
+        <th class="id">Email & Name</th>
```

Comment:
Maybe it doesn't but the [Internet seems to think it does](http://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp). Up to you.